src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -2641,6 +2641,24 @@ void ExecutorWorker::cancelAllOngoingJobs(bool reschedule /*= false*/) {
             _opList.push_front(_jobToSyncOpMap[job->jobId()]->id());
         }
     }
+
+    const auto startWaiting = std::chrono::system_clock::now();
+    while (!_ongoingJobs.empty()) {
+        if (std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now() - startWaiting).count() > 70) {
+            LOG_SYNCPAL_ERROR(
+                    _logger,
+                    "Timeout while waiting for jobs to finish (>70s), we will continue without waiting. It may cause a crash "
+                    "if the job ever finishes  [3.6.5 QUICK FIX]");
+            SentryHandler::instance()->captureMessage(
```

Comment:
So, there are two messages sent to Sentry: (1) one from `LOG_SYNCPAL_ERROR`, (2) the other from `captureMessage()`. 
Is this intended?